pkg/netutils/subnet_allocator.go

```diff
@@ -18,8 +18,9 @@ func NewSubnetAllocator(network string, capacity uint, inUse []string) (*SubnetA
 	}
 
 	netMaskSize, _ := netIP.Mask.Size()
+	fmt.Printf("Subnet prefix length is %d bits.\n", netMaskSize)
```

Comment:
Before I rebased this PR, there had been a `fmt.Println(netMaskSize)` that this commit deleted before adding the more explicit `fmt.Printf` statement.  I can delete this `fmt.Printf` if you believe it is superfluous.